app/controllers/posts_controller.rb

```diff
@@ -57,11 +57,20 @@ def set_post
   end
 
   def post_params
-    params.require(:post).permit(:caption, :tag_list, photos: [])
+    params.require(:post).permit(
+      :caption,
+      photos: [],
```

Comment:
Is this valid syntax in a permit? There's a failing Travis spec related to invalid syntax and I'm not familiar with setting a default this way